lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -115,7 +113,8 @@ def workspace_did_change_watched_files(changes)
             next
           end
 
-          entries = T.must(@index).entries_for(change[:uri])
+          entries = @index #: as !nil
+            .entries_for(change[:uri])
```

Comment:
Would this work instead?

```suggestion
          entries = @index&.entries_for(change[:uri])
```

Reply:
Or if the index is expected to be not nil (since we `T.must` it) I'd do `index = @index #: as !nil` at the beginning of the method.

Reply:
I went with the local variable 👍